custom.sample.php

```diff
@@ -15,16 +16,206 @@
 // do you own stuff in this file
 // we will never overrite this file
 
+/**
+ * API Request Caching
+ *
+ *  Use server-side caching to store API request's as JSON at a set 
+ *  interval, rather than each pageload.
+ * 
+ * @arg Argument description and usage info
+ */
+
+//Crypto prices, change variable according to your needs
+$coin='litecoin'; // bitcoin,litecoin,ethereum,zcash
+$price='USD'; //BTC,USD,EUR
+$link='https://api.coinmarketcap.com/v1/ticker/'.$coin.'?convert='.$price;
+$data=file_get_contents($link);
+$json=json_decode($data);
+$usd=$json[0]->price_usd;
```

Comment:
wring variable naming - hardcoded to "USD" - because its an API an in Line 29/30 you tell the User he can Switch the Currenc y;)

Reply:
Ah it should be $currency?

Reply:
How about $fiatCurrency

Reply:
what is the meaning of that? ;)
In the first Moment i think of a car from italy (Sorry!) ;)